packages/neon/neon/lib/src/models/app_implementation.dart

```diff
@@ -33,6 +36,18 @@ abstract class AppImplementation<T extends Bloc, R extends NextcloudAppOptions>
   @mustBeOverridden
   R get options;
 
+  /// Checks if the app is supported on the server of the [account].
+  ///
+  /// A return value of `null` means that it can not be known if the app is supported.
+  /// This is the case for apps that depend on the server version like files.
+  /// The server support is handled differently.
+  ///
+  /// The first value of the record is the supported status and the second value is the supported minimum version.
+  FutureOr<(bool, String?)?> isSupported(
```

Comment:
```suggestion
  FutureOr<(bool isSupported, String? minimumVersion)?> isSupported(
```

Reply:
Ahhh, stupid me forgot that you can name them :)